src/main/java/ch/zhaw/unicalc/pdfGenerator/Service/OfferInvoicePdf.java

```diff
@@ -0,0 +1,329 @@
+        table.addCell(totalDiscount);
+        Double toSubtract = Math.floor(total * offerRequest.getDiscount()) / 100;
+        Cell totalDisountComputed = new Cell(1, 1)
+                .add(new Paragraph("-" + toSubtract + " CHF"))
+                .setFontSize(9)
+                .setBorder(null)
+                .setPadding(0)
+                .setTextAlignment(TextAlignment.RIGHT);
+        table.addCell(totalDisountComputed);
+        table.addCell(emptySpace);
+        Cell nettoString = new Cell(1, 1)
+                .add(new Paragraph("Netto"))
+                .setFontSize(9)
+                .setBorderLeft(null)
+                .setBorderBottom(null)
+                .setBorderRight(null)
+                .setPadding(0)
+                .setTextAlignment(TextAlignment.RIGHT);
+        table.addCell(nettoString);
+        Double netto = total - toSubtract;
+        Cell nettoDouble = new Cell(1, 2)
+                .add(new Paragraph(netto + " CHF"))
+                .setFontSize(9)
+                .setBorderLeft(null)
+                .setBorderBottom(null)
+                .setBorderRight(null)
+                .setPadding(0)
+                .setTextAlignment(TextAlignment.RIGHT);
+        table.addCell(nettoDouble);
+        table.addCell(emptySpace);
+
+        Cell mwstString = new Cell(1, 1)
+                .add(new Paragraph("MWST"))
+                .setFontSize(9)
+                .setBorder(null)
+                .setPadding(0)
+                .setTextAlignment(TextAlignment.RIGHT);
+        table.addCell(mwstString);
+
+        Cell mwstDisount = new Cell(1, 1)
+                .add(new Paragraph("7.7%"))
+                .setFontSize(9)
+                .setBorder(null)
+                .setPadding(0)
+                .setTextAlignment(TextAlignment.RIGHT);
+        table.addCell(mwstDisount);
+        Cell mwstNumber = new Cell(1, 1)
+                .add(new Paragraph(Math.floor((netto + 0.077) * 100) / 100  + " CHF"))
+                .setFontSize(9)
+                .setBorder(null)
+                .setPadding(0)
+                .setTextAlignment(TextAlignment.RIGHT);
+        table.addCell(mwstNumber);
+        table.addCell(emptySpace);
+        Cell finalAmountString = new Cell(1, 1)
+                .add(new Paragraph("Gesamtbetrag"))
+                .setFontSize(9)
+                .setBorderLeft(null)
+                .setBorderRight(null)
+                .setPadding(0)
+                .setTextAlignment(TextAlignment.RIGHT);
+        table.addCell(finalAmountString);
+        Cell finalAmountNumber = new Cell(1, 2)
+                .add(new Paragraph(Math.floor((netto + 1.077) * 100) / 100  + " CHF"))
+                .setFontSize(9)
+                .setBorderLeft(null)
+                .setBorderRight(null)
+                .setPadding(0)
+                .setTextAlignment(TextAlignment.RIGHT);
+        table.addCell(finalAmountNumber);
+    }
+
+}
```

Comment:
Functions inside of this are pretty big, probably need to refactor this into smaller chunks that follow [SRP](https://en.wikipedia.org/wiki/Single-responsibility_principle) later.